src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {
+    String getColor();
+
+    void draw();
+}

```

Comment:
violated SRP principle. put `String getColor();` into Figure abstract class, and create separate interface `Drawable` for `void draw();`
```suggestion
```